### PR TITLE
[Automation] - Add logging v2 install test

### PR DESCRIPTION
**Pull request by izaac**
This tests the installation of the `rancher-logging` (Logging v2) chart using the v1 API.

Based off: #29425

### CODE REVIEW

File: tests/validation/tests/v3_api/test_v2_logging.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    cluster_provider = cluster_detail["cluster"]["provider"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # check if CIS is already installed and uninstall the app
```
Comment (sowmyav27) [inline tests/validation/tests/v3_api/test_v2_logging.py:51]: comment says CIS..

**Review by sowmyav27 — CHANGES_REQUESTED** [review]

**Review by sowmyav27 — APPROVED** [review]
LGTM

File: tests/validation/tests/v3_api/test_v2_logging.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    # install logging v2 app
+    logging_charts["chartName"] = CHART_NAME
+    logging_charts["releaseName"] = CHART_NAME
+    logging_charts["values"]["additionalLoggingSources"] = {cluster_provider: {"enabled": True}}
+    install_v2_app(client, rancher_repo, logging_charts, CHART_NAME, ns)
+
+
+@pytest.fixture(scope='module', autouse="True")
+def create_project_client(request):
+    client, cluster_detail["cluster"] = get_user_client_and_cluster()
```
Comment (jiaqiluo) [inline tests/validation/tests/v3_api/test_v2_logging.py:78]: Can you add a finalizer to uninstall the apps and delete the namespace at the end?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Comment by jiaqiluo** [comment]
@izaac  Can you rename both existing and new methods to follow the same pattern?
for example:
```
***_v2_app_*** -> ***_app_v2_***
***_v2_logging_*** -> ***_logging_v2_***
```

Or do you have any better idea or preferred pattern? 
cc @sowmyav27  @bmdepesa

**Comment by sowmyav27** [comment]
@jiaqiluo @izaac How about this

The file names - `test_v2_<chart>` like: `test_v2_cis_scan` or `test_v2_logging`

And the tests can be `test_v2_cis_install`, `test_v2_cis_create_clusterscan`, `test_v2_logging_install`. ?

I can the existing ones in CIS too in the next PR.

File: tests/validation/tests/v3_api/test_v2_logging.py (hunk 2 above)
Comment (izaac) [reply tests/validation/tests/v3_api/test_v2_logging.py:78]: I am following the same pattern from #29425 @jiaqiluo

**Comment by jiaqiluo** [comment]
> @jiaqiluo @izaac How about this
> 
> The file names - `test_v2_<chart>` like: `test_v2_cis_scan.py` or `test_v2_logging.py`. So that just `test_v2` --> will invoke all charts ?
> 
> And the tests can be `test_v2_cis_install`, `test_v2_cis_create_clusterscan`, `test_v2_logging_install`. ?
> 
> I can change the existing ones in CIS too in the next PR.

This sounds good to me.

All other methods for v2 apps should also follow the same pattern.

**Comment by izaac** [comment]
Closing, this will be re-implemented using the v1 API automation.

**Pull request closed without merging**